newIDE/app/src/EventsSheet/EventsTree/Renderers/GroupEvent.js

```diff
@@ -95,6 +95,9 @@ export default class GroupEvent extends React.Component<EventRendererProps, *> {
           }
         }}
         tabIndex={0}
+        id={`${groupEvent.getName()}-${
```

Comment:
It seems a bit dangerous to use the name. If we want to use templates in different languages, this wouldn't work. Should you use the index like you did for the events?

Reply:
fixed